polls/templates/polls/auth_flow.html

```diff
@@ -3,9 +3,7 @@
 {% block body-block %}
 
 <div class="mt-5 text-center">
-  {% if user.is_locked %}
```

Comment:
Why is the user lock functionality removed?

Reply:
Was it changed in another PR

Reply:
it's because he added the `CheckUserMixin` for pages. Now the Thank you page can be used for locked users.